### PR TITLE
Final result - last details fixed

### PR DESCRIPTION
**Pull request by Romain-Constant**

### CODE REVIEW

File: frontend/src/components/inmemoriam/InMemoriam.module.css [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+  justify-content: center;
+  margin-bottom: 20px;
+  padding: 10px 20px;
+}
+
+.badgeMission {
+  max-height: 140px;
+  width: auto;
+
+  margin-bottom: 20px;
```
Comment (Sax593) [inline frontend/src/components/inmemoriam/InMemoriam.module.css:108]: Unecessary space

File: frontend/src/components/inmemoriam/astronautCard/AstronautCard.module.css [hunk 2]
```diff
@@ -0,0 +1,53 @@
+.astroCard {
+  background-color: rgba(0, 0, 0, 0.404);
+  width: 7rem;
+  height: 7rem;
+
+  display: flex;
```
Comment (Sax593) [inline frontend/src/components/inmemoriam/astronautCard/AstronautCard.module.css:5]: unecessary space

File: frontend/src/components/nextLaunch/NextLaunchContainer.jsx [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+              ) : null}
+            </div>
+          </div>
+        </div>
+        <div className={styles.rightButtonContainer}>
+          <FaAngleRight
+            className={styles.button}
+            onClick={() => nextLaunch()}
+          />
+        </div>{" "}
```
Comment (Sax593) [inline frontend/src/components/nextLaunch/NextLaunchContainer.jsx]: {" "} unecessary

File: package.json [hunk 4]
```diff
@@ -29,5 +29,12 @@
     "concurrently": "^7.6.0",
     "cross-env": "^7.0.3",
     "husky": "^8.0.3"
+  },
+  "dependencies": {
+    "dayjs": "^1.11.7",
+    "dotenv": "^16.0.3",
+    "prop-types": "^15.8.1",
+    "react-icons": "^4.8.0",
+    "react-router-dom": "^6.10.0"
```
Comment (Sax593) [inline package.json]: It's front end package, please intall this in frontend package.json

**Review by Sax593 — CHANGES_REQUESTED** [review]

**Comment by Romain-Constant** [comment]
Changes done !

**Review by Sax593 — APPROVED** [review]

**Review by JocelynDeloose — APPROVED** [review]

**Pull request merged**